frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-item-details.tsx

```diff
@@ -58,36 +51,40 @@ type CapabilityLevelProps = {
 
 export const OperatorHubItemDetails: React.SFC<OperatorHubItemDetailsProps> = ({
   item,
-  closeOverlay,
   namespace,
 }) => {
   if (!item) {
     return null;
   }
   const {
-    name,
     installed,
-    iconClass,
-    imgUrl,
     provider,
     providerType,
+    obj,
     longDescription,
     description,
     version,
     repository,
     containerImage,
     createdAt,
     support,
-    catalogSource,
-    catalogSourceNamespace,
     capabilityLevel,
   } = item;
   const notAvailable = <span className="properties-side-panel-pf-property-label">N/A</span>;
 
+  const ibmSupportLink = _.get(obj, [
+    'status',
+    'channels',
+    '0',
+    'currentCSVDesc',
+    'annotations',
+    'marketplace.openshift.io/support-workflow',
+  ]);
```

Comment:
This should be read where the other CSV annotations are

https://github.com/spadgett/console/blob/bd4bf53545273a950023660b8482fc3ab60a6914/frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-page.tsx#L42-L50